README.md

```diff
@@ -31,22 +31,46 @@ EOF
 oc create -f ssh-secret.yml
 ```
 
+The CI pipeline supports a custom private registries where bundle and test
+index are pushed. To allow a private registry access user need to create
+auth secret - .dockerconfigjson with registry username and access token.
+
+```bash
+# Download an access token and create a secret
+
+cat << EOF > redhat-isv-redhat-isv-robot-secret.yml
```

Comment:
Do we really need to repeat "redhat-isv" twice here and elsewhere?

Reply:
The file was generated by quay.io when generating a token and this is the only example of how to use it so I guess it is no big deal.